### ISSUE TITLE
Code is not properly formatted.

### ISSUE THREAD

**Issue by jamilxt** [meheedihasaan/quiz-portal#36]
You can try using - Spotless to format the code automatically while building the project or before the commit. 

Ref: https://github.com/diffplug/spotless

**Comment by meheedihasaan** [comment]
@jamilxt bhai, I have added this in my pom.xml file. Let me know one thing - suppose I have two unformatted class. Now I want to commit the changes of that classes. If I don't build my project, then what will happen? Will the unformatted code be committed? Assume that I have committed the changes without building the project. I have done some changes and now want to commit again. This time I build the project and commit again. Now, what will happen? The previous two classes will be formatted and included with in the recent changes to commit? Isn't it conflicted?
To avoid this conflict, do we really need to build the project before committing?
